lib/ts/recipe/passwordless/components/themes/signInUp/emailForm.tsx

```diff
@@ -26,7 +26,11 @@ import type { SignInUpEmailFormProps } from "../../../types";
 
 export const EmailForm = withOverride(
     "PasswordlessEmailForm",
-    function PasswordlessEmailForm(props: SignInUpEmailFormProps): JSX.Element {
+    function PasswordlessEmailForm(
+        props: SignInUpEmailFormProps & {
+            footer?: JSX.Element;
```

Comment:
- It is because it's re-used by MFA
- Overriding the footers can be done separately
- There are cases where the two footers would need to be different (e.g.: showing some kind of disclaimer before signing up that isn't required on a secondary factor).